public/app/features/alerting/unified/api/buildInfo.ts

```diff
@@ -129,10 +140,20 @@ export async function discoverDataSourceFeatures(dsSettings: {
     };
   }
 
-  // @todo remove
-  throw Error('Unknown prometheus type!');
+  // @todo remove?
+  throw Error('Unknown prometheus flavor!');
 }
 
+const prometheusClientVersionGreaterOrEqualTo = (clientVersion: string | undefined, targetVersion: string) => {
```

Comment:
This is terrible and will be replaced with [semver](https://www.npmjs.com/package/semver)